docs/atlascli/command/atlas-streams-connections-update.txt

```diff
@@ -0,0 +1,110 @@
+.. _atlas-streams-connections-update:
+
+================================
+atlas streams connections update
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Modify the details of the specified connection within your Atlas Stream Processing instance.
+
+To use this command, you must authenticate with a user account or an API key that has the Project Owner role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas streams connections update <connectionName> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - connectionName
+     - string
+     - true
+     - Name of the connection.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -f, --file
+     - string
+     - true
+     - Path to a JSON configuration file that defines an Atlas Stream Processing connection.
+   * - -h, --help
+     - 
+     - false
+     - help for update
+   * - -i, --instance
+     - string
+     - true
+     - The name of your Atlas Stream Processing instance.
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Output
+------
+
+If the command succeeds, the CLI returns output similar to the following sample. Values in brackets represent your values.
+
+.. code-block::
+
+   Connection <Name> updated.
+   
+
+Examples
+--------
+
+.. code-block::
+
+   # update an Atlas Stream Processing connection:
+   atlas streams connection update kafkaprod -i test01 -f kafkaConfig.json
```

Comment:
Should this have --instance as a paramter

Reply:
-i is the shorthand for --instance, but I'll make it more verbose

Reply:
Resolved in [82c2186](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/82c2186c49ac36d4c34bfb1057ac3be1bb66941d)